cmd/derper/cert.go

```diff
@@ -76,6 +98,18 @@ func NewManualCertManager(certdir, hostname string) (certProvider, error) {
 	if err := x509Cert.VerifyHostname(hostname); err != nil {
 		return nil, fmt.Errorf("cert invalid for hostname %q: %w", hostname, err)
 	}
+	if _, err := netip.ParseAddr(hostname); err == nil {
```

Comment:
Could we parse it once and re-use here and line 85.

Minor optimization but not required.

Reply:
done